notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>typo on <code>Pandas</code> (capital <code>P</code>).</p><p> </p><p>Rather than "single column version", say something like "each column in a DataFrame is a Series".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>I'll convert all Pandas references with capital P to lowercase in all of the tutorials</p>

Reply:
<p>'each column in a DataFrame is a Series' -&gt; indedd, adjusted</p>